contrib/smtp_proxy/filters/network/source/smtp_session.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+#include <cstdint>
+
+#include "source/common/common/logger.h"
+
+#include "contrib/smtp_proxy/filters/network/source/smtp_transaction.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace SmtpProxy {
+
+class SmtpSession {
+public:
+  enum class State {
+    CONNECTION_REQUEST = 0,
```

Comment:
Change enums to camel case. For enums it should start with capital letter and follow with small letters. Like:
ConnectionRequest
ConnectionSuccess
etc.

Reply:
Done